ci/olm.sh

```diff
@@ -1,5 +1,17 @@
 cat > ci/olm.yaml <<EOF_CAT
```

Comment:
ok so this is the equiuvelent of 

https://github.com/openstack-k8s-operators/telemetry-operator/blob/acd9c11a60caf6ff96d31bb56e290975eb94d785/ci/create-coo-subscription.yaml#L2

is there a reason we are not takeign a similar approach to them and doing this in bash instead of ansible

https://github.com/openstack-k8s-operators/telemetry-operator/blob/acd9c11a60caf6ff96d31bb56e290975eb94d785/ci/create-coo-subscription-playbook.yaml

Reply:
Actually it came from a request from @raukadah to make it also work in the development workflow using make targets. I think it's a good idea.

Reply:
ok if there is a reasons to diverge like that we totally can.

i just didn't want use to have a different approach if there was not explicit reason to do so.